server/container/core/src/main/java/org/apache/james/server/core/MimeMessageCopyOnWriteProxy.java

```diff
@@ -41,14 +42,28 @@
 
 import org.apache.james.lifecycle.api.Disposable;
 import org.apache.james.lifecycle.api.LifecycleUtil;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * This object wraps a "possibly shared" MimeMessage tracking copies and
  * automatically cloning it (if shared) when a write operation is invoked.
  */
 public class MimeMessageCopyOnWriteProxy extends MimeMessage implements Disposable {
     public static class ContentTypeCleaner {
+        private static final Logger LOGGER = LoggerFactory.getLogger(ContentTypeCleaner.class);
+        private static final Pattern REGEX = Pattern.compile("^[\\w\\-]+/[\\w\\-]+");
+
         public static String cleanContentType(MimePart mimePart, String contentType) {
+            if (contentType == null) {
+                return null;
+            }
+
+            if (REGEX.matcher(contentType).find()) {
+                return contentType;
+            }
+
+            LOGGER.warn("Can not parse: " + contentType);
```

Comment:
maybe a bit more context could be welcomed? Like `Can not parse content-type: `?